### PR TITLE
fix unexpected warning on self bookmark in included file

### PR DESCRIPTION
**Pull request by 928PJY**
https://ceapex.visualstudio.com/Engineering/_workitems/edit/76318

### CODE REVIEW

File: docs/specs/resolve.yml [hunk 1]
```diff
@@ -49,6 +49,20 @@ outputs:
...
+# Links to self bookmarks in the included file
+inputs:
+  docfx.yml: |
+    files: docs/a.md
+  docs/a.md: |
+    # heading
+    [!INCLUDE[](includes/b.md)]
+  docs/includes/b.md: |
+    ## test
+    Link to [self](#test)
```
Comment (OsmondJiang) [inline docs/specs/resolve.yml]: add one new line 
`Link to [parent](#heading)`?

**Comment by live1206** [comment]
As we offline discussed, do we support bookmark in inclusion referencing to the parent file?

**Comment by 928PJY** [comment]
yes, we support it, they can reference the parent file with explicit path instead self bookmark,

---
In reply to: [476936095](https://github.com/dotnet/docfx/pull/4247#issuecomment-476936095) [](ancestors = 476936095)

File: docs/specs/resolve.yml [hunk 2]
```diff
@@ -49,6 +49,26 @@ outputs:
...
+    ## test
+    Link to [valiad self](#test)
+    Link to [valiad parent](../a.md/#heading)
+    Link to [invalid self](#heading1)
+    Link to [invalid parent](../a.md/#heading2)
+outputs:
+  docs/a.json: |
+    { "content": "<h2 id=\"test\">test</h2>\n<p>Link to <a href=\"#test\">valiad self</a>\nLink to <a href=\"a#heading\">valiad parent</a>\nLink to <a href=\"#heading1\">invalid self</a>\nLink to <a href=\"a#heading2\">invalid parent</a></p>" }
+  build.log: |
+    ["suggestion","internal-bookmark-not-found","Cannot find bookmark '#heading2' in 'docs/includes/b.md', did you mean '#heading'?","docs/includes/b.md","",5,9]
```
Comment (OsmondJiang) [inline docs/specs/resolve.yml]: it's the #heading2 is treated as internal bookmark, then could we add a case `Link to [valiad parent](#heading)`

Comment (928PJY) [reply docs/specs/resolve.yml]: we don't recommend the user to use selfbookmark to reference the  parent

---
In reply to: [269406727](https://github.com/dotnet/docfx/pull/4247#discussion_r269406727) [](ancestors = 269406727)

**Pull request merged**